### DISCUSSION TITLE
Talos on Macs?

### DISCUSSION THREAD

**Discussion by mcdent** [siderolabs/talos#6923] [Q&A]
Hey
I have 3 or 4 Intel Mac's I am thinking of setting up as a kubernetes cluster for homelab type stuff. Is there any reason I can't install Talos on to these?
I'd prefer to do bare metal installs on these. 
Thanks.

**Comment by mcdent** [comment]
Ok, so answering my own question, partly! I managed to boot the ISO image on one Mac, with Thunderbolt 2 and the Apple TB to Ethernet adaptor.
I managed to boot it on a newer 2017 MacBook Pro with only Thunderbolt 3/USB ports. However it appears it would not detect the network adaptor (or the port) correctly as it just sat at the prompt trying to sync the time, which continued.,
I do have another couple of Mac's I will try with but I was hoping to use the newer ones, faster proc etc.

**Reply by dhess** [reply]
I'm running Talos v1.4 on several 2017 iMac Pros, but I need to patch the installer so that it resizes the EFI partition to at least 200MiB, or else the machines won't boot. Otherwise, they seem to work well.

Recently I've been trying to use NVMe enclosures via the Thunderbolt ports to add additional storage, but the drives don't show up. Have you had any success with that, and if so, did you need to tweak the kernel command line, or install any additional kernel modules?

**Reply by frezbo** [reply]
Starting with Talos 1.5, the UKI based UEFI only Talos assets should have 300Meg as the default EFI size.
